tests/foreman/ui/test_settings.py

```diff
@@ -199,9 +200,9 @@ def test_positive_register_hostname_and_cvs_dependencies_update(session, setting
 
 @pytest.mark.tier3
 @pytest.mark.parametrize('setting_update', ['login_text'], indirect=True)
-def test_positive_update_login_page_footer_text_with_long_string(session, setting_update):
+def test_positive_update_login_page_footer_text_with_long_and_empty_string(session, setting_update):
```

Comment:
@adarshdubey-star test name looks too specific. Can we just trim it and keep something like `test_positive_update_login_page_footer_text`

Reply:
Sure!